src/components/table/__spec__.js

```diff
@@ -931,6 +930,21 @@ describe('Table', () => {
       expect(parent.className).toEqual('carbon-table foo');
     });
 
+    it('renders a caption tag when a caption prop is given', () => {
+      let wrapper = render(
```

Comment:
I can't remember to be honest. I suspect it didn't work with `shallow` when I tried it, but it works fine with `shallow` now.

Also, I've just compared using `render` to using `shallow`, and `shallow` was 4-5 times faster! So I've gone with `shallow` 👍